crawler.c

```diff
@@ -293,7 +293,8 @@ static int lru_crawler_poll(crawler_client_t *c) {
 
     if (to_poll[0].revents & POLLIN) {
         char buf[1];
-        int res = read(c->sfd, buf, 1);
+        int res = c->c == NULL ? read(c->sfd, buf, 1) :
```

Comment:
when would c->c be NULL?

Reply:
Coming to think of it, I don't believe it can be.